libr/core/cmd_helps.help

```diff
@@ -0,0 +1,231 @@
+#include <r_cmd.h>
+
+// root helps
+
+static const RCmdDescHelp system_help = {
```

Comment:
BTW, later if we want to make this "easier" to contribute to for non-C devs or we just want to avoid all the boilerplate code, we may consider generating this file from a json or something and converting it to this format at build-time. This way people will be able to update the help without having to worry about C code. WDYT?

Reply:
+1